README.md

```diff
@@ -49,7 +49,7 @@ import { output } from "./awaiting.mjs";
 export function outputPlusValue(value) { return output + value }
```

Comment:
Not sure how this should be formatted. Should I put a semicolon here?

```suggestion
export function outputPlusValue(value) { return output + value; }
```

(As a no-semi person, I don’t think it’s necessary, but as you’re using semicolons, I’m not sure)

Reply:
yes, one belongs there.